commit faeeac629d2416d6a9d1516f61b983437a3d1558 (refs/changes/30/2330/1)
Author: Tim Kordas <tim.kordas@joyent.com>
Date:   2017-08-03T12:18:07-07:00 (2 years, 2 months ago)
    
    MANTA-3364: add haproxy-stat back as a dependency, change path appropriately

diff --git a/alarm_metadata/probe_templates/loadbalancer.yaml b/alarm_metadata/probe_templates/loadbalancer.yaml
index b1548b1..a917eca 100644
--- a/alarm_metadata/probe_templates/loadbalancer.yaml
+++ b/alarm_metadata/probe_templates/loadbalancer.yaml
@@ -133,7 +133,7 @@
         -
             type: cmd
             config:
-                cmd: "test -n $(/opt/smartdc/muppet/build/node/bin/node /opt/smartdc/muppet/node_modules/.bin/haproxystat showStat /tmp/haproxy  | /opt/smartdc/muppet/build/node/bin/node -e 's=\"\"; process.stdin.resume(); process.stdin.on(\"data\",function(c){s+=c}); process.stdin.on(\"end\",function(){o=eval(\"(\"+s+\")\");console.log(JSON.stringify(o)); });' | /usr/bin/json  -c 'this.type == \"backend\" && !/stats/.test(this.pxname) && act == 0' -a pxname)"
+                cmd: "test -n $(/opt/smartdc/manta-deployment/build/node/bin/node /opt/smartdc/manta-deployment/node_modules/.bin/haproxystat showStat /tmp/haproxy  | /opt/smartdc/manta-deployment/build/node/bin/node -e 's=\"\"; process.stdin.resume(); process.stdin.on(\"data\",function(c){s+=c}); process.stdin.on(\"end\",function(){o=eval(\"(\"+s+\")\");console.log(JSON.stringify(o)); });' | /usr/bin/json  -c 'this.type == \"backend\" && !/stats/.test(this.pxname) && act == 0' -a pxname)"
                 interval: 30
                 threshold: 2
                 period: 120
diff --git a/package.json b/package.json
index 6df79ca..8dc1915 100644
--- a/package.json
+++ b/package.json
@@ -14,6 +14,7 @@
         "extsprintf": "1.3.0",
         "fash": "2.3.1",
         "forkexec": "0.1.0",
+        "haproxy-stat" : "0.1.0",
         "hogan.js": "2.0.0",
         "imgapi-cli": "git+https://github.com/joyent/sdc-imgapi-cli.git#65bba66818",
         "jsprim": "^1.4.0",
